File: ChallengeBackend/src/main/java/br/com/savedra/challengebackend/model/Banner.java
package br.com.savedra.challengebackend.model;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import org.springframework.data.annotation.Id;
import org.springframework.data.mongodb.core.mapping.Document;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Document(collection = "banners")
public class Banner {
    @Id
    private String id;
    private String title;
    private String description;
    private String imageUrl;
    private String segment;
}

File: ChallengeCRM/app/src/main/java/br/com/savedra/challengecrm/model/Banner.kt
package br.com.savedra.challengecrm.model

/**
 * Modelo de Banner para o Android.
 */
data class Banner(
  val id: String? = null,
  val title: String? = "",
  val description: String? = "",
  val imageUrl: String? = "",
  val segment: String? = ""
)
